### ISSUE TITLE
instructor note additions

### ISSUE THREAD

**Issue by ChristinaLK** [datacarpentry/sql-ecology-lesson#314]
from the 2021-01-05 community discussion, these themes arose, which could be added to the instructor notes or the introductory episode. 

- important to get to joins
- important to talk about getting data in/out of databases
- this example presents one example of database use, but maybe include something at the beginning/end that talks more about how SQL can fit into research workflows (looking up publically available data, intermediate processing before using python/R, etc.). Lesson should explain why a database (and not excel) for example. That info is there, but could be conveyed more clearly. 
- Also maybe mention/emphasize that this is a widely used paradigm in certain fields

**Comment by sjcorey** [comment]
I agree that it is very important to get to joins. I suggest that joins should include an instructor discussion of the types of joins that are possible with DB Browser for SQLite, while other joins are not. As a learner, a visual comparison of the types of table joins would help tremendously.

**Comment by ChristinaLK** [comment]
I agree that a visual comparison would be super helpful!
